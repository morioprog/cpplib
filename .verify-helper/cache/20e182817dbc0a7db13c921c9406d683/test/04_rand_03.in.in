99
1 29
0
0
1 16
2 93 94
2 31 90
0
0
3 15 34 47
1 39
1 44
1 36
1 46
0
0
0
0
1 71
1 69
1 25
0
2 12 83
2 61 98
1 87
0
0
0
0
0
3 43 51 80
1 72
0
1 11
0
2 28 38
2 77 84
0
0
0
0
1 57
1 37
0
1 54
1 91
1 2
1 53
0
0
2 59 66
0
0
2 1 63
3 23 68 95
2 3 89
0
0
1 97
2 65 73
0
0
1 41
0
0
3 60 85 92
1 6
1 56
1 20
1 5
0
2 14 19
0
1 81
1 24
6 4 10 33 42 82 86
1 9
0
0
1 76
2 55 74
1 50
2 49 64
4 17 35 40 48
2 27 70
1 58
0
0
0
3 7 8 67
5 18 52 62 75 79
2 26 96
3 13 32 78
0
1 88
3 21 22 30
0
0
1 45
0
100
1 82
0 72 33
1 12
1 73
0 98 96
0 33 98
0 67 55
0 40 88
1 17
0 84 64
1 42
0 16 87
0 97 91
0 15 77
1 85
1 9
1 46
0 18 37
1 83
0 14 29
1 36
1 1
1 11
0 64 38
0 60 23
0 70 58
0 66 27
0 34 52
0 80 29
0 68 89
1 50
0 6 77
0 46 32
0 57 100
0 85 53
0 80 36
1 14
1 63
1 65
1 85
1 76
1 47
1 0
0 94 75
0 59 26
0 25 13
0 78 22
0 34 40
1 87
0 17 58
1 11
1 67
0 67 53
1 14
1 62
0 78 74
0 36 52
1 62
0 91 73
0 14 60
1 18
1 51
1 16
0 45 53
0 40 70
0 5 84
0 65 1
1 69
0 27 90
1 89
1 78
0 70 13
1 14
1 65
1 43
1 86
1 17
1 30
1 5
0 37 36
1 94
0 69 90
0 61 74
1 0
1 54
0 86 34
1 96
0 50 94
1 35
1 21
0 58 72
1 89
1 30
0 71 91
0 1 1
1 15
1 16
1 55
0 95 45
0 92 52
