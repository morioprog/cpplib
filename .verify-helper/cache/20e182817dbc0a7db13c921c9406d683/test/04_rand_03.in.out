0
0
0
0
0
33
0
0
0
0
0
0
29
87
0
64
124
0
0
0
107
0
55
162
0
0
37
0
87
37
0
169
235
65
0
0
58
75
280
75
0
0
280
0
75
0
75
77
87
0
